### ISSUE TITLE
About learning rate

### ISSUE THREAD

**Issue by zwh6879a** [akamaster/pytorch_resnet_cifar10#31]
Greetings, I found in your code that the starting learning rage was set to be 0.1, which is too large for the resnet to improve, and in the first 100 epochs the accuracy of the model seemed to be 10%. Is this designed on purpose?
